database/schema.sql

```diff
@@ -34,9 +34,7 @@ CREATE TABLE constellation (
 CREATE TABLE body (
```

Comment:
This endeavor proves to be immensely advantageous in meticulously aligning the Entity-Relationship Diagram (ERD) with the intricate and nuanced requirements articulated by our esteemed stakeholders. By doing so, we not only enhance clarity and coherence in our communications but also foster a deeper understanding of the project's structural framework, ultimately paving the way for a more collaborative and informed decision-making process.